### ISSUE TITLE
Train a network after document embeddings step

### ISSUE THREAD

**Issue by benjisympa** [flairNLP/flair#1170]
Hello, I have a question about document embeddings.
I do this code to get one embedding per sentence :

```
sentences_embeddings = Variable(torch.zeros((len(X_), self.config['taille_embedding']))).float()
for idx, s in enumerate(X_):
    sentence = Sentence(s)
    # embed the sentence with our document embedding
    document_embeddings.embed(sentence)
    sentences_embeddings[idx, :] = sentence.get_embedding()
sentences_embeddings = sentences_embeddings.unsqueeze(1)# (B,D) -> (L,B,D)
sentences_embeddings = sentences_embeddings.to(self.device)
```

After this, I put theses sentences embeddings to another LSTM network, but the training is too much fast, I suspect that the document embeddings are fixed and not trained together with my network. Document embeddings have require gradient set to True normally, so I don't understand why my LSTM and the LSTM of the document embeddings class are not train together ?
I don't know if I'm enough clear ?

Thanks so much for your answer.
